.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  semi: false,
+  trailingComma: 'all',
+  singleQuote: true,
+  printWidth: 80,
+  tabWidth: 2,
```

Comment:
As discussed let's try without config or just empty object. Also this should not be committed to the project AFAIK. You should simply have some config for your editor. There should not be any need for code changes in particular repositories.